### PR TITLE
Add a few enhancements to morphometry plotting

### PR DESCRIPTION
**Pull request by mwaskom**
- Allow for a custom colormap
- Allow for custom min/max
- Plot bivariate measures (curv and sulc) with the midpoint at 0
- Compute the min and max of the colormap robustly
- Allow morphometry colorbar to be optional

### CODE REVIEW

**Comment by Eric89GXL** [comment]
I'll fix Travis, just a sec

File: surfer/viz.py [hunk 1]
```diff
@@ -1163,6 +1163,7 @@ def remove_labels(self, labels=None, hemi=None):
                 ll.remove()
 
     def add_morphometry(self, measure, grayscale=False, hemi=None,
+                        colormap=None, min=None, max=None, colorbar=True,
```
Comment (Eric89GXL) [inline surfer/viz.py]: Looks like you added more options than are documented. Also, maybe these should go after `remove_existing` to ensure backward compat?

**Comment by mwaskom** [comment]
OK that addresses the argument order and documentation.

**Comment by mwaskom** [comment]
Should I rebase on master to pick up the pep8 fixes?

**Comment by Eric89GXL** [comment]
No, you don't need to, they get picked up by the merge testing Travis does

**Comment by mwaskom** [comment]
Any other comments here?

**Comment by Eric89GXL** [comment]
No, I meant to merge after seeing green but forgot. Thanks for the reminder

**Comment by mwaskom** [comment]
No worries; don't think these changes will conflict with the config_opts stuff but figured better to do things in serial just to be safe.

**Pull request merged**